api/src/main/java/io/druid/data/input/impl/CSVParseSpec.java

```diff
@@ -35,26 +34,35 @@
 {
   private final String listDelimiter;
   private final List<String> columns;
+  private final boolean firstRowIsHeader;
 
   @JsonCreator
   public CSVParseSpec(
       @JsonProperty("timestampSpec") TimestampSpec timestampSpec,
       @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec,
       @JsonProperty("listDelimiter") String listDelimiter,
-      @JsonProperty("columns") List<String> columns
+      @JsonProperty("columns") List<String> columns,
+      @JsonProperty("firstRowIsHeader") boolean firstRowIsHeader
```

Comment:
How about calling this `hasHeaderRow`? (It might not be the first row… maybe we'll have some "skip N rows" configs at some point)